tests/mock_network/churn.rs

```diff
@@ -72,14 +72,13 @@ fn remove_unresponsive_node() {
     // Sending some user events to create a sequence of observations.
     let mut responded = 0;
     for i in 0..UNRESPONSIVE_WINDOW {
-        let event: Vec<_> = rng.gen_iter().take(100).collect();
+        let event: Vec<_> = gen_vec(&mut rng, 100);
```

Comment:
I think the `Vec<_>` qualification is redundant now (this applies to other places in this file).